teamapps-ux/src/main/java/org/teamapps/ux/session/SessionContext.java

```diff
@@ -72,6 +72,18 @@
 
 import static org.teamapps.common.util.ExceptionUtil.softenExceptions;
 
+/**
+ * Provides access to several aspects of a Ui-Session and allows sending commands and updates to the client.
+ * A UI-session starts upon opening the website and ends when leaving or reloading it. The Communication between client
+ * and server happens via websocket.
+ * Only one thread can interact with a Ui-Session at a time. Therefore, all interactions with the session context from
```

Comment:
Single thread access is important for mainly two reasons:
- to guarantee a certain order of events on the client and server side (most important: both process changes in the **same** order)
- because none UI Components is thread-safe